gef.py

```diff
@@ -95,17 +96,22 @@
     # Compat Py2/3 hacks
     def range(*args):
         """Replace range() builtin with an iterator version."""
-        if len(args) < 1:
-            raise TypeError()
-        start, end, step = 0, args[0], 1
-        if len(args) == 2: start, end = args
-        if len(args) == 3: start, end, step = args
+        if len(args) == 0:
```

Comment:
To answer your question in https://github.com/hugsy/gef/pull/480#discussion_r343742689:
I want to make this check explicit.